service/storage_proxy.cc

```diff
@@ -400,8 +400,6 @@ class storage_proxy::remote {
     }
 
     future<> send_truncate_blocking(sstring keyspace, sstring cfname, std::optional<std::chrono::milliseconds> timeout_in_ms) {
```

Comment:
> fixing https://github.com/scylladb/scylladb/issues/11087 is the perfect

Should you add fixed https://github.com/scylladb/scylladb/issues/11087 then?

Reply:
It's in the PR description, but could be added to the commit message too.
(and remember about `scylladb/scylladb#` prefix)

Reply:
I added Fixes